apiserver/plane/app/views/analytic/advance.py

```
@@ -120,9 +119,25 @@ def get_overview_data(self) -> Dict[str, Dict[str, int]]:
             ),
         }
 
-
-    def get_work_items_stats(self) -> Dict[str, Dict[str, int]]:
-        base_queryset = Issue.issue_objects.filter(**self.filters["base_filters"])
+    def get_work_items_stats(
+        self, cycle_id=None, module_id=None
+    ) -> Dict[str, Dict[str, int]]:
+        """
+        Returns work item stats for the workspace, or filtered by cycle_id or module_id if provided.
```

Ruff (0.11.9)

Line too long (100 > 88)

(E501)